src/main/java/mekanism/common/util/MekanismUtils.java

```diff
@@ -767,7 +767,7 @@ public static Map<Fluid, FluidInDetails> getFluidsIn(Player player, UnaryOperato
     }
 
     public static void veinMineArea(IEnergyContainer energyContainer, Level world, BlockPos pos, ServerPlayer player, ItemStack stack, Item usedTool,
-          Collection<BlockPos> found, Function<Float, FloatingLong> destroyEnergyFunction, DoubleUnaryOperator distanceMultiplier, BlockState sourceState) {
+                                    Collection<BlockPos> found, Function<Float, FloatingLong> destroyEnergyFunction, BiFunction<Double, Boolean, Double> distanceMultiplier, BlockState sourceState) {
```

Comment:
Random indentation being added, but I also feel like we might as well just add a functional interface for `(double, boolean) -> double` so as to avoid having to box three different things while calculating the distance multiplier. I also feel like it might be a bit better to actually to have it be `(double, BlockState) -> double` so that we only lookup if it is a disassembler ore in places where needed.

Reply:
I like changing to block state. But I kinda like using the bifunction, how set are you on switching that? Also I should probably let you know the link for the intellij style sheet is down so I've kinda been guessing on some of the formatting.

Reply:
BiFunction is probably fine and also the style sheet should be here: https://github.com/mekanism/Mekanism/blob/1.18.x/docs/GoogleStyle4Indents.xml (is it linked to from somewhere else that we forgot to update the link at some point for?)

Reply:
https://github.com/mekanism/Mekanism/blob/template_update/docs/GoogleStyle4Indents.xml
was the link I was trying from the contributing guidelines page. It was giving a 404 file not found thing. Still is for me.

Reply:
Ah interesting I will make sure to update the contributing guidelines to point to the docs spot then.